src/porepy/numerics/fv/biot.py

```diff
@@ -75,10 +75,23 @@ def rhs_bound(self, g, data):
         matrices_m = data[pp.DISCRETIZATION_MATRICES][self.mechanics_keyword]
         matrices_f = data[pp.DISCRETIZATION_MATRICES][self.flow_keyword]
 
+        bound_stress = matrices_m["bound_stress"]
+        bound_flux = matrices_f["bound_flux"]
+        if bound_stress.shape[0] != g.dim * g.num_faces:
```

Comment:
Same comment as for mpfa/mpsa: If bcs by subface are permissible, this must be documented. See also below.